pkg/util/throttle.go

```diff
@@ -26,9 +26,12 @@ type RateLimiter interface {
 	Accept()
 	// Stop stops the rate limiter, subsequent calls to CanAccept will return false
 	Stop()
+	// Saturation returns a percentage number which describes how saturated
+	// this rate limiter is.
```

Comment:
I initially thought I agree with @gmarek  but after more thinking about it, I think that I don't agree.

I think that the reason behind having this metric is to expose how many operations we are still able to process. And we are not going to look into a single value of the metric, but its graph over time.

So if the saturation is going up, it's a clear signal that there are more requests that we can actually handle (which is obviously a bad sign).

I think that we should leave as is (and potentially add another metric (something similar to what @gmarek suggested) in the future).